### DISCUSSION TITLE
tests: develop and write tests for steps/update_stay_duration.py

### DISCUSSION THREAD

**Discussion by anujsinha3** [uw-ssec/MAWpy#100] [Ideas]
The Update Stay Duration code has been refactored from using CSV files to the pandas data frame.

The task is to develop and write tests for the code at: src/mawpy/steps/update_stay_duration.py

To write tests:

create a folder called 'steps' under the 'tests' folder
In the above folder, create a file called 'test_update_stay_duration.py'
define test functions corresponding to functions in src/mawpy/steps/update_stay_duration.py

**Comment by anujsinha3** [comment]
@gracejia513 @Anurag19101996

**Comment by qzchen-uw** [comment]
@anujsinha3 did you provide some sample tests for Grace and Anurag to follow?

**Comment by anujsinha3** [comment]
This is in review but it can be used for soft reference: https://github.com/uw-ssec/MAWpy/pull/56/files#diff-492e435e9c31b6f3f74e8cafa31aee5a467cafb00f0e93a55b8f1e1b0ae516c8
